src/s1/s1/static/js/Packages/S1/Workbench/ProposalEditColumn.html

```diff
@@ -3,7 +3,7 @@
         <div class="moving-column-menu-nav">
             <a
                 class="moving-column-menu-nav-button"
-                data-ng-href="{{ processUrl | adhResourceUrl:meeting }}">{{ 'TR__CLOSE' | translate }} X</a>
+                data-ng-href="{{ proposalUrl | adhResourceUrl }}">{{ 'TR__CLOSE' | translate }} X</a>
```

Comment:
I thought we wanted the same behavior of "cancel" in other proposal-edit-view, e.g. of IdeaColl. But the wording and style are "close", so I should revert this.